docs/src/pages/index.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Nebari Homepage
+---
+
+<!-- Copy for Nebari Landing page. See Firma file: https://www.figma.com/file/ccOw2sxgjpEUjrOgKqVXRq/Nebari-website?node-id=809%3A3439 -->
+
+## Hero
+
+Tagline: An opinionated open sourc eplatform for Data Science and research teams. Built for scale, designed for collaboration.
+
+## Why choose Nebari?
+
+Nebari provides a unified platform for all users to collaborate on data science and research projects.
+It is designed to be flexible and extensible, and built entirely on open source infrastructure.
+Nebari is vendor-and-cloud-agnostic, and is designed and developed by and for the data science community.
+
+<!-- Need to highlight 6 features -->
+<!-- Added here left-to-right, top-to-bottom -->
```

Comment:
Should be in the format of:

1. Feature (couple of words max)
   One-liner about said feature 
   
  See [Dask site](https://www.dask.org/) for an example

Reply:
Suggestion "Power of Dask" -> "Dask powered"

Reply:
I'm done updating the copy!

> Suggestion "Power of Dask" -> "Dask powered"
@trallard Oh, nice! I went with "Powerd by Dask", but I also like "Dask powered" -- feel free to select the one you prefer!

Reply:
I like Dask powered best!

Reply:
Integrated @pavithraes wording + some minor edits